crates/bevy_core_pipeline/src/core_2d/camera_2d.rs

```diff
@@ -37,6 +38,8 @@ pub struct Camera2dBundle {
     pub deband_dither: DebandDither,
     pub main_texture_usages: CameraMainTextureUsages,
     pub msaa: Msaa,
+    /// Marker component that indicates that its entity needs to be Synchronized to the render world
```

Comment:
For here and all other comments, please do not capitalize "Synchronized" in the middle of the sentence.